app/inpututils.cpp

```diff
@@ -476,7 +476,7 @@ QString InputUtils::geometryLengthAsString( const QgsGeometry &geometry )
 {
   QgsDistanceArea distanceArea;
   distanceArea.setEllipsoid( QStringLiteral( "WGS84" ) );
-  distanceArea.setSourceCrs( QgsCoordinateReferenceSystem::fromEpsgId( 4326 ), QgsCoordinateTransformContext() );
+  distanceArea.setSourceCrs( PositionKit::positionCRS(), QgsCoordinateTransformContext() );
```

Comment:
I am not sure if this is the right thing to do - the distance measurement only uses 2D coordinates, so using 3D coordinate system may be a bit confusing and suggest we're doing 3D measurements even we do not. I would suggest to revert this...